client/templates/topic/topicInfoItemList.js

```diff
@@ -145,9 +145,9 @@ Template.topicInfoItemList.helpers({
         const tmpl = Template.instance();
         if (itemId && itemId === tmpl.data.items[index]._id) {
             Session.set('topicInfoItem.triggerAddDetailsForItem', null);
-            Meteor.setTimeout(() => {
+            /*Meteor.setTimeout(() => {
                 addNewDetails(tmpl, index).catch(handleError);
-            }, 1300); // we need this delay otherwise the input field will be made hidden immediately
+            }, 100); // we need this delay otherwise the input field will be made hidden immediately*/
```

Comment:
Please do not commit commented out code to github. If code is not needed anymore it should be removed - if we need it lateron we have old github commit to dig it out agein.

In fact the whole helper method should be removed. From: client/templates/topic/topicInfoItemList.js remove this helper: `triggerAddDetails: function(index)`
After this, remove the call to the helper in: client/templates/topic/topicInfoItemList.html  => remove this line:
`{{triggerAddDetails @index}}`